app/src/main/res/layout/fragment_time_sheet.xml

```diff
@@ -10,120 +11,395 @@
+                            />
+                        <ImageButton
+                            android:layout_width="@dimen/dp_60"
+                            android:layout_height="@dimen/dp_20"
+                            android:layout_alignParentEnd="true"
+                            android:layout_alignParentRight="true"
+                            android:background="@drawable/md_transparent"
+                            android:onClick="@{()-> viewModel.onNextMonth()}"
+                            android:scaleType="centerCrop"
+                            android:tint="@color/colorPrimary"
+                            app:srcCompat="@drawable/ic_menu_right"
+                            bind:ignore="ContentDescription"
+                            />
+                    </RelativeLayout>
+
+                </FrameLayout>
+
+            </android.support.design.widget.CollapsingToolbarLayout>
+
+        </android.support.design.widget.AppBarLayout>
+
+        <android.support.v4.widget.NestedScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            app:layout_behavior="@string/appbar_scrolling_view_behavior"
+            >
+            <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginLeft="@dimen/dp_5"
-                android:layout_marginRight="@dimen/dp_5"
-                android:background="@android:color/white"
-                android:visibility="@{viewModel.showInformation ? View.VISIBLE : View.GONE}"
-                app:cardCornerRadius="@dimen/dp_5"
+                android:layout_height="match_parent"
+                android:orientation="vertical"
                 >
-                <LinearLayout
+                <android.support.v7.widget.CardView
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:orientation="vertical"
+                    android:layout_marginBottom="@dimen/dp_5"
+                    android:layout_marginLeft="@dimen/dp_5"
+                    android:layout_marginRight="@dimen/dp_5"
+                    android:background="@android:color/white"
+                    android:visibility="@{viewModel.showInformation ? View.VISIBLE : View.GONE}"
                     >
-                    <TextView
+                    <LinearLayout
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:gravity="center"
+                        android:orientation="vertical"
                         android:padding="@dimen/dp_10"
-                        android:text="@{viewModel.date}"
-                        android:textSize="@dimen/sp_18"
-                        />
+                        >
+                        <LinearLayout
```

Comment:
WARNING: [UseCompoundDrawables] This tag and its children can be replaced by one `<TextView/>` and a compound drawable
 A `LinearLayout` which contains an `ImageView` and a `TextView` can be more efficiently handled as a compound drawable (a single TextView, using the `drawableTop`, `drawableLeft`, `drawableRight` and/or `drawableBottom` attributes to draw one or more images adjacent to the text).

If the two widgets are offset from each other with margins, this can be replaced with a `drawablePadding` attribute.

There's a lint quickfix to perform this conversion in the Eclipse plugin.